app/templates/components/provider-carousel.hbs

```diff
@@ -5,21 +5,16 @@
             <div class={{if (not index) "item active" "item"}}>
                 {{#each slide as |provider index|}}
                     <div class={{if (not index) (concat "col-sm-2 " columnOffset) "col-sm-2"}}>
-                        {{#if lightLogo}}
-                            <a href="/preprints/{{provider.id}}"
-                                title={{provider.name}}
-                                onclick={{action 'click' 'link' (concat 'Index - Provider Carousel - ' provider.name)}}
-                                >
-                                <div class="provider-logo provider-{{provider.id}}" style="background-image: url('{{providerAsset provider.id 'wide_white.png'}}')"></div>
-                            </a>
-                        {{else}}
-                            <a href="/preprints/{{provider.id}}/discover"
-                                title={{provider.name}}
-                                onclick={{action 'click' 'link' (concat 'Discover - Provider Carousel ' provider.name)}}
+                        <a href="{{brandedPreprintUrl provider type=lightLogo}}"
```

Comment:
Instead of `type` we should use a better name for this parameter. What might even be better is to just pass in the path to append and leave the logic of deciding what to append in the template.